src/sas/cli.py

```diff
@@ -103,6 +107,11 @@ def parse_cli(argv: list[str]) -> argparse.Namespace:
         opts.args = rest
     return opts
 
+def is_script(filename):
+    # allow .py.txt as the script file name for those with mail systems that filter .py files
+    filename = Path(filename)
+    return filename.suffix in ('.py', '.py.txt')
```

Comment:
`Path("path/script/py.txt").suffix` is `.txt` not `.py.txt`.

Again, but with testing:
```python
def is_script(filename: str | Path) -> bool:
    filename = str(filename)
    return any(filename.endswith(target) for target in (".py", ".py.txt"))
```